src/stories/Customization.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import LinkTo from '@storybook/addon-links/react';
+
+<Meta title="Customization" />
+
+## Customization
+
+Some design aspects of the visualizations, such as the font-size of the tick labels, the color of the title, the color of the axes, etc. are **controlled by CSS**.
+We provide sane defaults that should be sufficient to start with, but at some point
+you might like to tweak a few things so the visualizations integrate better with your app's design.
+
+### Inherited font properties
+
+Font properties (`font-size`, `font-family`, etc.) of textual elements (tick labels, plot title, etc.) are **inherited by default**, so tweaking them all at once is easy.
```

Comment:
What properties are inherited in addition to `font-size` and `font-family` ?

Reply:
`color`, `line-height`, `font-size`, `font-weight` ...

Reply:
https://stackoverflow.com/a/5612360/758806

Reply:
I might just add `color`.

Reply:
I've added `font-weight` for good measure.